src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Ball {
+    private Colors color;
+    private int number;
+
+    public Ball(Colors color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball's color is " + color + " "
+                + "ball's number is " + number + "\n";
```

Comment:
Avoid using line breaks in the return statement. It's better to return the complete string in one line.